src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -1282,10 +1282,10 @@ s32 EnTa_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* po
         this->stateFlags &= ~TALON_STATE_FLAG_SUPPRESS_ROCKING_ANIM;
     } else if ((limbIndex == ENTA_LIMB_CHEST) || (limbIndex == ENTA_LIMB_LEFT_ARM) ||
                (limbIndex == ENTA_LIMB_RIGHT_ARM)) {
-        s32 limbIdx50 = limbIndex * 50;
+        s32 overridePerLimb = limbIndex * LIMB_OVERRIDE_PER_I;
 
```

Comment:
Im unsure what `LIMB_OVERRIDE_PER_I` is supposed to mean. I also dont think that `overridePerLimb` is the right choice of name here, but need to follow what `LIMB_OVERRIDE_PER_I` is to be able to offer a suggestion

Reply:
with the port of of MM naming i changed the set of defines as such:
```
#define FIDGET_BASE_Y 0x814
#define FIDGET_BASE_Z 0x940
#define FIDGET_PER_I 0x32
#define FIDGET_SCALE 200.0f
```

`Actor_UpdateFidgetTables` doesn't have a notion of limbs, so `FIDGET_PER_LIMB` might be not ideal too.

generally fidgeting formula goes as follows:
```
rot += sine((BASE + i * PER_I) * frame) * SCALE
```

maybe something like
```
#define FIDGET_ADD_Y 0x814
#define FIDGET_ADD_Z 0x940
#define FIDGET_MUL_I 0x32
#define FIDGET_SCALE 200.0f
```
that will do?